### ISSUE TITLE
Translations doesn´t effect some menus/words

### ISSUE THREAD

**Issue by KojoFI** [OpenViX/enigma2#588]
I found that even some phrases are translated, they are not shown in menu´s
Words like "Event info", "Timer menu", "EPG search", "Do nothing" and that HDMI-IN stuff

EDIT:HDMI-stuff was misunderstanding, others are relevant.

**Comment by Huevos** [comment]
Please be specific. Exact steps to reproduce each individual issue.

Where did you get the .po file?

**Comment by KojoFI** [comment]
I updated my box with 5.4 and fetched .po file from there.

I dont know how to be more specific. I´ve translated using PoEDIT and there those words are translated, but when I upload translation to my box, they stay untranslated.

There were almos 300 fuzzy finnish translations. I updated those that I knew existed and were not changed.
Then I removed rest of those fuzzy translations so those will be easier to find in the next release.
